### PR TITLE
feat: add sequelize-ts-example

### PR DESCRIPTION
**Pull request by whxaxes**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change

Add egg-sequelize-ts example, deps on: https://github.com/eggjs/egg-sequelize/pull/73

### CODE REVIEW

**Comment by whxaxes** [comment]
CI 过了，@atian25 看一下？

File: sequelize-ts/app/controller/post.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+import { Controller } from 'egg';
+
+export default class PostController extends Controller {
```
Comment (fengmk2) [inline sequelize-ts/app/controller/post.ts]: 这个文件没看出 ts 跟 js 有啥差别。。。

File: sequelize-ts/database/migrations/20180813112934-user.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const { INTEGER, DATE, STRING } = Sequelize;
+    await queryInterface.createTable('users', {
+      id: { type: INTEGER, primaryKey: true, autoIncrement: true },
+      name: STRING(30),
+      age: INTEGER,
+      created_at: DATE,
+      updated_at: DATE,
```
Comment (fengmk2) [inline sequelize-ts/database/migrations/20180813112934-user.js]: 日期这样默认值到秒，需要DATE(6) 默认到毫秒

File: sequelize-ts/app/controller/post.ts (hunk 1 above)
Comment (whxaxes) [reply sequelize-ts/app/controller/post.ts]: 对，controller 的写法 ts 跟 js 区别不大

File: sequelize-ts/database/migrations/20180813112934-user.js (hunk 2 above)
Comment (whxaxes) [reply sequelize-ts/database/migrations/20180813112934-user.js]: 好，我改一下

**Pull request merged**